config/clients/java/template/Java/libraries/native/api.mustache

```diff
@@ -0,0 +1,591 @@
+{{>licenseInfo}}
+package {{package}};
+
+import {{invokerPackage}}.ApiClient;
+import {{invokerPackage}}.ApiException;
+import {{invokerPackage}}.ApiResponse;
+import {{invokerPackage}}.Pair;
+
```

Comment:
This file seems like it is repeating api above

Reply:
Thanks -- Deleted in [3dbbaf7](https://github.com/openfga/sdk-generator/pull/158/commits/3dbbaf752f74a14f3535e48df92b4a934822699d)